### PR TITLE
Overcoming Grammatical errors

### PR DESCRIPTION
**Pull request by Mohit-Nain**
one typo needs correction

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/permissions/organization_roles.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ weight = 2
 
 # Organization Roles
 
-Users can be belong to one or more organizations. A user's organization membership is tied to a role that defines what the user is allowed to do
+Users be belong to one or more organizations. A user's organization membership is tied to a role that defines what the user is allowed to do
```
Comment (peterholmberg) [inline docs/sources/permissions/organization_roles.md]: I think the correct wording should be

```suggestion
Users can belong to one or more organizations. A user's organization membership is tied to a role that defines what the user is allowed to do
```

**Review by peterholmberg — CHANGES_REQUESTED** [review]
Left a suggestion, can you also update the PR title to be more informative. 

https://github.com/grafana/grafana/blob/master/contribute/create-pull-request.md#formatting-guidelines

**Comment by Mohit-Nain** [comment]
Made the required changes and corrected the PR title as well.

**Review by aknuds1 — APPROVED** [review]
LGTM

**Pull request merged**